dvc/repo/experiments/queue/base.py

```diff
@@ -420,12 +420,14 @@ def _stash_exp(  # noqa: PLR0915, C901
                     run_env[DVC_EXP_NAME] = name
                     if resume_rev:
                         run_env[DVCLIVE_RESUME] = "1"
-                    studio_repo_url = get_studio_repo_url()
+
+                    studio_token, studio_repo_url = get_studio_token_and_repo_url()
```

Comment:
Handling the token here is not needed for the fix but I anticipate that we eventually would want to make this work getting the token from other sources beyond the env var